media-libs/vidstab/vidstab-1.1.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_PN="vid.stab"
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+SRC_URI="https://github.com/georgmartius/${MY_PN}/archive/v${PV}.tar.gz
+	-> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND=">=media-video/ffmpeg-3.2.4-r1"
+DEPEND="dev-util/cmake
```

Comment:
do not depend on cmake

Reply:
fixed (since inheriting cmake-utils does it, I guess)